tests/cross/src/test/scala/tests/tokens/SemanticTokensSuite.scala

```diff
@@ -0,0 +1,180 @@
+package tests.tokens
+
+import java.net.URI
+
+import scala.meta.internal.jdk.CollectionConverters._
+import scala.meta.internal.metals.CompilerVirtualFileParams
+
+import munit.Location
+import munit.TestOptions
+import tests.BasePCSuite
+import tests.TestSemanticTokens
+
+class SemanticTokensSuite extends BasePCSuite {
+
+
```

Comment:
```suggestion
  override protected def ignoreScalaVersion: Option[IgnoreScalaVersion] = Some(
    IgnoreScala3
  )
```
this should ignore running tests on Scala 3, which is not yet implemented.